tarsier/tag_utils.ts

```diff
@@ -0,0 +1,208 @@
+// noinspection JSUnusedGlobalSymbols
+interface Window {
+  tagifyWebpage: (tagLeafTexts?: boolean) => { [key: number]: string };
+  removeTags: () => void;
+}
+
+const elIsClean = (el: HTMLElement) => {
+  if (el.style && el.style.display === "none") return false;
+  if (el.hidden) return false;
+  // @ts-ignore
```

Comment:
🤔

Reply:
Is HTMLElement the right type here then?

Reply:
theres a lot of them, didnt want to deal with it rn